src/v0/destinations/braze/docs/businesslogic.md

```diff
@@ -0,0 +1,165 @@
+# Braze Business Logic and Mappings
+
+## Overview
+
+This document outlines the business logic and mappings used in the Braze destination integration. It covers how RudderStack events are mapped to Braze's API format and the special handling for various event types.
+
+## Event Mappings
```

Comment:
For all of these lets give reference to the braze APIs we are calling for these events